docs/reference/index-modules/translog.asciidoc

```diff
@@ -1,41 +1,42 @@
 [[index-modules-translog]]
 == Translog
 
-Changes to Lucene are only persisted to disk during a Lucene commit,
-which is a relatively heavy operation and so cannot be performed after every
-index or delete operation. Changes that happen after one commit and before another
-will be lost in the event of process exit or HW failure.
+Changes to Lucene are only persisted to disk during a Lucene commit, which is a
+relatively heavy operation and so cannot be performed after every index or
+delete operation. Changes that happen after one commit and before another will
+be lost in the event of process exit or hardware failure.
```

Comment:
instead of saying "lost" (which implies maybe total loss) maybe say - "will be removed from the index by Lucene in the event of..."

Reply:
++

I reworded the subsequent paragraph too as it didn't follow after making that change.